### PR TITLE
HDDS-10118. hdds-rocks-native fails to build with Java11+

### PR DESCRIPTION
**Pull request by raju-balpande**
## What changes were proposed in this pull request?
 Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.0.2:copy-dependencies (copy-jars) on project hdds-rocks-native: Artifact has not been packaged yet. When used on reactor artifact, copy should be executed after packaging: see MDEP-187.

I found the file names mismatch in the the referred directory

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-10118

## How was this patch tested?
Tested with command line 
alias mvn11="JAVA_HOME=/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home && mvn"
mvn11 clean install -DskipTests -e -DskipShade -X -Drocks_tools_native

### CODE REVIEW

**Comment by adoroszlai** [comment]
It looks like the original issue has been fixed (probably by HDDS-10149) since it was reported.  The patch fixes another problem @raju-balpande found:

```
[INFO] --- maven-dependency-plugin:3.6.1:copy-dependencies (copy-jars) @ hdds-rocks-native ---
[INFO] Copying hdds-managed-rocksdb-1.5.0-SNAPSHOT.jar to hadoop-hdds/rocks-native/target/dependency/hdds-managed-rocksdb-1.5.0-SNAPSHOT.jar
[INFO] Copying hdds-common-1.5.0-SNAPSHOT.jar to hadoop-hdds/rocks-native/target/dependency/hdds-common-1.5.0-SNAPSHOT.jar
[INFO] Copying hdds-hadoop-dependency-client-1.5.0-SNAPSHOT.jar to hadoop-hdds/rocks-native/target/dependency/hdds-hadoop-dependency-client-1.5.0-SNAPSHOT.jar
[INFO] Copying hadoop-annotations-3.3.6.jar to hadoop-hdds/rocks-native/target/dependency/hadoop-annotations-3.3.6.jar
...
[INFO] --- exec-maven-plugin:3.1.1:exec (javach) @ hdds-rocks-native ---
error: file not found: hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileReaderIterator.java
Usage: javac <options> <source files>
use --help for a list of possible options
19:17:56,727 [ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 2 (Exit value: 2)
```

File: hadoop-hdds/rocks-native/pom.xml [hunk 1]
```diff
@@ -393,7 +393,7 @@
                     <argument>-h</argument>
                     <argument>${project.build.directory}/native/javah</argument>
                     <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileReader.java</argument>
-                    <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileReaderIterator.java</argument>
+                    <argument>${project.basedir}/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRawSSTFileIterator.java</argument>
```
Comment (swamirishi) [inline hadoop-hdds/rocks-native/pom.xml:396]: Oh my bad ! Missed this in my previous commit. @raju-balpande Thanks for pointing this out.

**Review by swamirishi — APPROVED** [review]
@raju-balpande Thanks for the patch LGTM

**Comment by adoroszlai** [comment]
Thanks @raju-balpande for the patch, @swamirishi for the review.

**Pull request merged**